index.html

```diff
@@ -25,7 +25,40 @@
                 <img class="menu__icon" src="./images/Header/icons/hamburger.svg" alt="Menu icon" />
             </div>
         </header>
-    </div>
+
+        <div class="hero container">
+            <div class="hero__content">
+                <div class="hero__text">
+                <h2 class="hero__welcome">Hello</h2>
+                <h1 class="hero__title">I’m <span class="bold">Sarah Jackson</span></h1>
+                <p class="hero__desc">A UX / UI designer, product designer, content creator and creative director.</p>
+                <p class="hero__desc2">Of course, there’s so much more to me than just a few fancy titles. Scroll down and get to know me,</p>
```

Comment:
Maybe here you haven't chosen the proper elements in terms of semantics. Right now your elements look like this:

```
h2
h1
p
p
```
 and they all have different sizes. Maybe it would be more appropriate to do something like this:
```
span
h1
h2
p
```

Reply:
Also you can simplify the classes like this:
```
span.hero-label
h1.hero-title
h2.hero-subtitle
p.hero-text
```